stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiChatApi.kt

```diff
@@ -912,4 +925,12 @@ constructor(
     private fun <T : Any> noConnectionIdError(): ErrorCall<T> {
         return ErrorCall(coroutineScope, ChatError("setUser is either not called or not finished"))
     }
+
+    override fun <T : Any> postponeCallIfNeeded(shouldPostpone: Boolean, call: () -> Call<T>): Call<T> {
+        return callPostponeHelper.postponeCallIfNeeded(shouldPostpone, call)
+    }
+
+    override fun <T : Any> postponeCall(call: () -> Call<T>): Call<T> {
+        return callPostponeHelper.postponeCall(call)
+    }
```

Comment:
I think those method could be `private` and been used only from inside of this class.
This class should now when a call need to be postponed or not